***************
*** 3,15 ****
  start_timer = 0;
  side_b = scr_sideb_active();
  
! if (!side_b || global.plot < 63 || global.plot >= 91)
  {
      instance_destroy();
      exit;
  }
  
! if (!i_ex(obj_homealone_vent_loader))
  {
      var loader = instance_create(0, 0, obj_homealone_vent_loader);
      loader.load_room = 52;
--- 3,15 ----
  start_timer = 0;
  side_b = scr_sideb_active();
  
! if (false && (!side_b || global.plot < 63 || global.plot >= 91))
  {
      instance_destroy();
      exit;
  }
  
! if (false && !i_ex(obj_homealone_vent_loader))
  {
      var loader = instance_create(0, 0, obj_homealone_vent_loader);
      loader.load_room = 52;
***************
*** 61,67 ****
  drip_modifier = 30;
  drip_index = 0;
  drip_timer = 0;
! con = 0;
  auto_text = false;
  auto_text_buffer_time = 90;
  auto_text_buffer = auto_text_buffer_time;
--- 61,67 ----
  drip_modifier = 30;
  drip_index = 0;
  drip_timer = 0;
! con = 1;
  auto_text = false;
  auto_text_buffer_time = 90;
  auto_text_buffer = auto_text_buffer_time;
